flashinfer/xqa.py

```
@@ -423,7 +422,7 @@ def xqa_mla(
     assert k_cache.dtype == v_cache.dtype, "K and V cache must have the same dtype"
 
     if get_compute_capability(torch.device(device="cuda"))[0] not in [12]:
-        raise RuntimeError("XQA is only supported on SM120 GPUs")
+        raise RuntimeError("XQA MLA is only supported on SM120 GPUs")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)